.travis.yml

```diff
@@ -9,7 +9,7 @@ before_script:
   - rustup component add clippy-preview
 
 script:
-  - cargo test
+  - cargo test -- --test-threads=1
```

Comment:
@sile @yuezato 

It's better to limit the parallelism by `--test-threads=1` because build 22 might fail due to a multithreading issue.  In build 24, I didn't change the code but all tests passed.

build 22: https://travis-ci.org/frugalos/liberasurecode/jobs/451695510
build 24: https://travis-ci.org/frugalos/liberasurecode/builds/451704079

Reply:
And we must investigate any workaround to avoid probabilistic test failures...

Reply:
It is out of scope of this PR to deal with the above problem which is probably caused among creating an EC instance.
Just now, we merge this PR and will tackle it on anther PR.